public/pages/Monitors/containers/Monitors/Monitors.js

```diff
@@ -180,9 +180,12 @@ export default class Monitors extends Component {
 
   updateMonitor(item, update) {
     const { httpClient } = this.props;
-    const { id, version, monitor } = item;
```

Comment:
Let's call them seqNo and primaryTerm, the ifSeqNo/ifPrimaryTerm is just for the API query term. The actual values represent seqNo and primaryTerm.

Reply:
Sure. Will fix it part of next PR. This is how currently it is being used rest of the place. Will change it together.